.travis.yml

```diff
@@ -25,8 +25,8 @@ jobs:
     - os: linux
       env:
         - MB_PYTHON_VERSION=3.7
-        - NP_BUILD_DEP="numpy==1.16.5"
-        - NP_TEST_DEP="numpy==1.16.5"
+        - NP_BUILD_DEP="numpy==1.17.3"
+        - NP_TEST_DEP="numpy==1.17.3"
```

Comment:
a couple of lines below, we have `NP_BUILD_DEP="numpy==1.19.1"` for 3.8. Our min on master in pandas (i.e. 1,4) is 1.18.5. Any idea why there is a discrepancy there?

Reply:
nvm. it's probably for the aarch support.